ledger/complete/mtrie/flattener/trie_test.go

```diff
@@ -9,31 +9,28 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/onflow/flow-go/ledger"
-	"github.com/onflow/flow-go/ledger/common/utils"
 	"github.com/onflow/flow-go/ledger/complete/mtrie/flattener"
 	"github.com/onflow/flow-go/ledger/complete/mtrie/trie"
 )
 
 func TestTrieStoreAndLoad(t *testing.T) {
-
-	pathByteSize := 32
 	dir, err := ioutil.TempDir("", "test-mtrie-")
 	require.NoError(t, err)
 	defer os.RemoveAll(dir)
 
-	emptyTrie, err := trie.NewEmptyMTrie(pathByteSize)
+	emptyTrie := trie.NewEmptyMTrie()
 	require.NoError(t, err)
 
-	p1 := utils.PathByUint8(1)
-	v1 := utils.LightPayload8('A', 'a')
-	p2 := utils.PathByUint8(2)
-	v2 := utils.LightPayload8('B', 'b')
-	p3 := utils.PathByUint8(130)
-	v3 := utils.LightPayload8('C', 'c')
-	p4 := utils.PathByUint8(131)
-	v4 := utils.LightPayload8('D', 'd')
-	p5 := utils.PathByUint8(132)
-	v5 := utils.LightPayload8('E', 'e')
+	p1 := ledger.PathByUint8(1)
+	v1 := ledger.LightPayload8('A', 'a')
+	p2 := ledger.PathByUint8(2)
+	v2 := ledger.LightPayload8('B', 'b')
+	p3 := ledger.PathByUint8(130)
+	v3 := ledger.LightPayload8('C', 'c')
+	p4 := ledger.PathByUint8(131)
+	v4 := ledger.LightPayload8('D', 'd')
+	p5 := ledger.PathByUint8(132)
+	v5 := ledger.LightPayload8('E', 'e')
```

Comment:
Same comments as above exposing these test utils on the root package is not ideal, might result in mistakes by developers.

Reply:
Yep, I'll work on fixing this as stated here: https://github.com/onflow/flow-go/pull/514#discussion_r618034401